molecule/default/tests/test_cinder_volume.py

```diff
@@ -14,15 +14,22 @@
 
 
 os_pre = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
-          "-- bash -c '. /root/openrc ; openstack ")
+          "-- bash -c '. /root/openrc ; ")
```

Comment:
Should the `os_pre` variable in this test be renamed to `utility_container` to be consistent with the other tests in this commit?

Reply:
updated the variable to `utility_container`